index.js

```diff
@@ -60,12 +60,12 @@ module.exports = {
 }
 
 function createBot (options = {}) {
-  options.username = options.username ?? 'Player'
-  options.version = options.version ?? false
-  options.plugins = options.plugins ?? {}
-  options.hideErrors = options.hideErrors ?? true
-  options.logErrors = options.logErrors ?? true
-  options.loadInternalPlugins = options.loadInternalPlugins ?? true
+  options.username = options.username || 'Player'
+  options.version = options.version || false
+  options.plugins = options.plugins || {}
+  options.hideErrors = options.hideErrors || true
```

Comment:
Please read [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)
hideErrors, logErrors and loadInternalPlugins will be forever true